Sources/Options.swift

```diff
@@ -594,6 +594,16 @@ public enum ClosingParenPosition: String, CaseIterable {
     case `default`
 }
 
+public enum SwiftUIPropertiesSortMode: String, CaseIterable {
+    /// Sorts all SwiftUI dynamic properties alphabetically
+    case alphabetize
+    /// Sorts SwiftUI dynamic properties by grouping all dynamic properties of the same type by using the first time each property appears
+    /// as the sort order.
+    case firstAppearanceSort
```

Comment:
Other arguments seem to use kebab-case rather than camelCase

```suggestion
    case firstAppearanceSort = "first-appearance-sort"
```